CHANGELOG.md

```diff
@@ -19,6 +21,8 @@
 
 * Compatibility with python 3.6
 
+* Fix a problem with read_carroucell()
```

Comment:
You should put this in a new section (0.2.7) as 0.2.6 is already published.

Reply:
done !